### PR TITLE
[ISSUE #8417]add some test cases for org.apache.rocketmq.common.AclConfig

### LINKED ISSUE #8417: [Enhancement] Add test cases for org.apache.rocketmq.common.AclConfig
### Before Creating the Enhancement Request

- [X] I have confirmed that this should be classified as an enhancement rather than a bug/feature.

### Summary

Add test cases for common module, org.apache.rocketmq.common.AclConfig

### Motivation

for issue https://github.com/apache/rocketmq/issues/8262 , add test cases for common module

### Describe the Solution You'd Like

create new test case by using tongyilingma and add comments.

### Describe Alternatives You've Considered

NULL

### Additional Context

With the ability of Tongyi Lingcode, the code test coverage of all modules of the supplementary project reached more than 50%.

### PR DESCRIPTION
**Pull request by TeFuirnever**
### Which Issue(s) This PR Fixes

[Enhancement] Add test cases for org.apache.rocketmq.common.AclConfig #8417

Fixes #8417

### Brief Description

add some test cases for org.apache.rocketmq.common.AclConfig.

### How Did You Test This Change?

run test case successfull.
image

### CODE REVIEW

**Comment by TeFuirnever** [comment]
for soc in https://github.com/apache/rocketmq/issues/8262 .

File: common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java
Comment (yx9o) [inline common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java:32]: Test classes do not require method comments.

Comment (TeFuirnever) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java:32]: Is it to remove all method level comments?

Comment (yx9o) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java:32]: Yes, it's not necessary in test class.

Comment (TeFuirnever) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java:32]: By the way, why does my check item show "7 workflows awaiting approval"

Comment (yx9o) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java:32]: Wait, it's first contribution, need the approval of the committer.

Comment (TeFuirnever) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java]: Thank you very much, I've modified it

Comment (TeFuirnever) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java]: Sorry to bother you again, is codecov-commenter triggered automatically or do you need to trigger it manually?
image

Comment (yx9o) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java]: ci automatically triggered.

**Review by yx9o — APPROVED** [review]

**Review by mxsm — APPROVED** [review]
LGTM

File: common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java
Comment (TeFuirnever) [reply common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java]: > ci automatically triggered.

Thank you very much

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**
